src/plot_api/plot_api.js

```diff
@@ -2124,6 +2126,7 @@ Plotly.update = function update(gd, traceUpdate, layoutUpdate, traces) {
             data: restyleSpecs.eventData,
             layout: relayoutSpecs.eventData
         });
+        gd.emit('plotly_plotmodified');
```

Comment:
Didn't know about it! Just need to add that event in a couple more places, but that should be perfect.